railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt

```diff
@@ -131,5 +131,17 @@
 # If your cookies can't yet be serialized to JSON, keep using `:marshal` for backward-compatibility.
 #
 # If you have configured the serializer elsewhere, you can remove this section of the file.
-# 
+#
 # See https://guides.rubyonrails.org/action_controller_overview.html#cookies for more information.
+
+# Change the return value of `ActionDispatch::Request#content_type` to the Content-Type header without modification.
+# Rails.application.config.action_dispatch.return_only_request_media_type_on_content_type = false
+
+# Active Storage `has_many_attached` relationships will default to replacing the current collection instead of appending to it.
+# Thus to support submitting an empty collection, when `multiple_file_field_include_hidden` is `true`
```

Comment:
```suggestion
# Thus, to support submitting an empty collection, the `include_hidden` option must be used with `file_field`. Setting `multiple_file_field_include_hidden` to true enables this option for all `file_field`s by default.
```

I don't quite like my suggestion, so definitely take it with a grain of salt. However, I do think it's important to mention what the config does, which is set `include_hidden` on `file_field`s.